dist/index.js

```diff
@@ -4948,13 +4952,26 @@ function getVersionFromSpec(spec, versions) {
       version =
         versionsMap[semver.maxSatisfying(Object.keys(versionsMap), rangeForMax)]
     } else {
-      version = versionsMap[spec]
+      version = versionsMap[maybeCoerced(spec)]
```

Comment:
We were missing this. We added, say, `1.13.2` for `1.13.2.1` but then searched for `1.13.2.1`, which wouldn't be found.